diff --git a/node_modules/react-notion-x/build/index.js b/node_modules/react-notion-x/build/index.js
index cbcb3b3..60d740c 100644
--- a/node_modules/react-notion-x/build/index.js
+++ b/node_modules/react-notion-x/build/index.js
@@ -976,31 +976,14 @@ var type_github_default = SvgTypeGitHub;
 
 // src/components/mention-preview-card.tsx
 import { jsx as jsx6, jsxs as jsxs4 } from "react/jsx-runtime";
-function capitalizeFirstLetter(str) {
-  if (!str) return "";
-  return str.charAt(0).toUpperCase() + str.slice(1);
-}
-function MentionPreviewCard({
-  owner,
-  lastUpdated,
-  externalImage,
-  title,
-  domain
-}) {
+function MentionPreviewCard(props) {
+  const { title, owner, domain } = props;
   return /* @__PURE__ */ jsxs4("div", { className: "notion-external-subtitle", children: [
-    externalImage && /* @__PURE__ */ jsxs4("div", { className: "notion-preview-card-domain-warp", children: [
-      /* @__PURE__ */ jsx6("div", { className: "notion-preview-card-logo", children: externalImage }),
-      /* @__PURE__ */ jsx6("div", { className: "notion-preview-card-domain", children: capitalizeFirstLetter(domain.split(".")[0]) })
-    ] }),
     /* @__PURE__ */ jsx6("div", { className: "notion-preview-card-title", children: title }),
     owner && /* @__PURE__ */ jsxs4("div", { className: "notion-external-subtitle-item", children: [
       /* @__PURE__ */ jsx6("div", { className: "notion-external-subtitle-item-name", children: "Owner" }),
       /* @__PURE__ */ jsx6("span", { className: "notion-external-subtitle-item-desc", children: owner })
     ] }),
-    lastUpdated && /* @__PURE__ */ jsxs4("div", { className: "notion-external-subtitle-item", children: [
-      /* @__PURE__ */ jsx6("div", { className: "notion-external-subtitle-item-name", children: "Updated" }),
-      /* @__PURE__ */ jsx6("span", { className: "notion-external-subtitle-item-desc", children: lastUpdated })
-    ] }),
     domain === "github.com" && /* @__PURE__ */ jsxs4("div", { className: "notion-preview-card-github-shields", children: [
       /* @__PURE__ */ jsx6(
         "img",
